lib/task_helpers/exports/customization_templates.rb

```diff
@@ -0,0 +1,33 @@
+module TaskHelpers
+  class Exports
+    class CustomizationTemplates
+      EXCLUDE_ATTRS = %i(created_at updated_at id pxe_image_type_id class).freeze
+      def export(options = {})
+        export_dir = options[:directory]
+
+        customization_templates = options[:all] ? CustomizationTemplate.all : CustomizationTemplate.where(:system => [false, nil])
+
+        customization_templates.order(:id).each do |customization_template|
+          $log.info("Exporting Customization Template: #{customization_template.name} (ID: #{customization_template.id})")
+
+          ct_hash = Exports.exclude_attributes(customization_template.to_model_hash, EXCLUDE_ATTRS)
+          ct_hash.merge!(pxe_image(customization_template.pxe_image_type))
+
+          image_type_name = ct_hash[:pxe_image_type][:name] ? ct_hash[:pxe_image_type][:name] : "Examples"
+          filename = Exports.safe_filename("#{image_type_name}-#{ct_hash[:name]}", options[:keep_spaces])
+          File.write("#{export_dir}/#{filename}.yaml", ct_hash.to_yaml)
```

Comment:
Should we be timestamping these filenames at all?

Reply:
I don't believe so. In many (probably most) cases these scripts are being used to maintain the .yaml files in an SCM, Adding a timestamp to the filename would defeat the ability to see modifications over time.

Reply:
Also, none of the other export scripts add a timestamp to the filename.